### ISSUE TITLE
Cudnn frontend sdpa wrong assertion in attention module with d_bias

### ISSUE THREAD

**Issue by Skylion007** [NVIDIA/cudnn-frontend#115]
**Describe the bug**

Cudnn_frontend asserts on d_bias, even when the d_bias isn't used. See this PyTorch issue: https://github.com/pytorch/pytorch/issues/137347 When upgrading the CUDNN_Frontend, code that used to work fine no longer works, and the assertion does not seem appropiate:

```
RuntimeError: cuDNN Frontend error: For cuDNN version below 9.5.0, dBias not support s_q/s_kv which aren't multiple of 64
```

https://github.com/pytorch/pytorch/issues/137347

**Expected behavior**
Previously working code would remain working. The assertion appears to be incorrect as the dbias isn't used: 

**System Environment (please complete the following information):**

- cudnn_frontend 1.7.0

**To Reproduce**
1. Update PyTorch and update the submodule to 1.7.0
2. Run the example in https://github.com/pytorch/pytorch/issues/137347
3. Watch the cudnn error propogate

**Additional context**

**Comment by Skylion007** [comment]
Ping @eqy

**Comment by eqy** [comment]
IIUC cuDNN is targeting a fix in a few days-week in an upcoming release

**Comment by Skylion007** [comment]
This should be fixed by 9.5.0
